src/components/CalendarMarkup/CalendarMarkup.jsx

```diff
@@ -0,0 +1,25 @@
+import { useContext } from 'react'
+import './CalendarMarkup.scss'
+import UsersContext from '../../contexts/UsersContext'
+
+const CalendarMarkup = () => {
+    const { user } = useContext(UsersContext)
+    const days = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri']
```

Comment:
Please pull the last changes from 'main' and move days and times to the /src/constants.js